io/net/network_test.py

```diff
@@ -138,12 +140,20 @@ def test_lro(self):
         '''
         ro_type = "lro"
         ro_type_full = "large-receive-offload"
-        path = '/sys/class/net/%s/device/uevent' % self.iface
+        bond_interface = False
+        path = '/sys/class/net/%s/uevent' % self.iface
         output = open(path, 'r').read()
         for line in output.splitlines():
-            if "OF_NAME" in line:
-                if 'vnic' in line.split('=')[-1]:
-                    self.cancel("Unsupported on vNIC")
+            if 'DEVTYPE' in line:
+                if 'bond' in line.split('=')[-1]:
+                    bond_interface = True
+        if not bond_interface:
+            path = '/sys/class/net/%s/device/uevent' % self.iface
+            output = open(path, 'r').read()
+            for line in output.splitlines():
+                if "OF_NAME" in line:
+                    if 'vnic' in line.split('=')[-1]:
+                        self.cancel("Unsupported on vNIC")
```

Comment:
@smruti77 Can you explain the need for this change in detail ?

Reply:
@narasimhan-v actually "'/sys/class/net/%s/device/uevent'" this is the path not available for bonding interface.that's why its getting error during scenario test run.so i added to check its bonding interface or not. if it is bonding interface it will not check for that path.that path is only to check vnic interface or not.

Reply:
So check if that path exists, and cancel for vnic. No need to check if it is bond interface or not.

Reply:
ok